agents-docs/quick-reference.md

```
@@ -1,56 +1,146 @@
-# Quick Reference — Unique Commands
-
-Commands NOT covered by skill files or `./scripts/validate.sh`.
+# Quick Reference Commands
 
 ## Build Performance
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)